gon.config.hcl

```diff
@@ -0,0 +1,10 @@
+source = ["dist/arduino_cli_osx_darwin_amd64/arduino-cli"]
+bundle_id = "cc.arduino.arduino-cli"
+
+sign {
+  application_identity = "Developer ID Application: ARDUINO SA (7KT7ZWMCJT)"
+}
+
+zip {
+  output_path = "arduino-cli.zip"
+}
```

Comment:
Since we are ignoring the output zip file, can't we just remove this section and not produce it in the first place?

Reply:
Yes you're right, I misread the documentation. I thought that at least one output format was mandatory :+1: